docs/readmes/basics/prerequisites.md

```diff
@@ -58,7 +58,7 @@ Development can occur from multiple OS's, where **macOS** and **Ubuntu** are **e
 ### Ubuntu
 
 1. Install the following tools
-   1. [Docker](https://docs.docker.com/engine/install/ubuntu/) and [Docker Compose](https://docs.docker.com/compose/install/)
+   1. [Docker and Docker Compose](https://docs.docker.com/engine/install/ubuntu/)
```

Comment:
Should the line 28 for MacOS also be adapted to be more specific? `   1. [Docker](https://www.docker.com) and Docker Compose`